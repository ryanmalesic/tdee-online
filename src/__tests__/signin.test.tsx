import React from 'react';
import renderer from 'react-test-renderer';

import SignIn from '../pages/signin';

it('renders signin unchanged', () => {
  const tree = renderer.create(<SignIn />).toJSON();
  expect(tree).toMatchSnapshot();
});
