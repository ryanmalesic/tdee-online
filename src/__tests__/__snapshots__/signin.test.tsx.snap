// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders signin unchanged 1`] = `
Array [
  <nav
    className="navbar is-fixed-top"
    role="navigation"
  >
    <div
      className="navbar-brand"
    >
      <a
        className="navbar-item"
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        <h1
          className="title"
        >
          TDEE Online
        </h1>
      </a>
      <a
        className="navbar-burger burger "
        data-target="navbarBasicExample"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          aria-hidden="true"
        />
        <span
          aria-hidden="true"
        />
        <span
          aria-hidden="true"
        />
      </a>
    </div>
    <div
      className="navbar-menu"
      id="navbarBasicExample"
    >
      <div
        className="navbar-start"
      >
        <a
          className="navbar-item"
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Home
        </a>
      </div>
      <div
        className="navbar-end"
      >
        <div
          className="navbar-item"
        >
          <div
            className="buttons"
          >
            <a
              className="button has-background-primary-dark"
              href="/signup"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              <strong>
                Sign up
              </strong>
            </a>
            <a
              className="button has-text-dark has-background-grey-lighter"
              href="/signin"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Sign in
            </a>
          </div>
        </div>
      </div>
    </div>
  </nav>,
  <section
    className="hero is-fullheight background"
  >
    <div
      className="hero-body"
    >
      <div
        className="container"
      >
        <div
          className="columns is-centered"
        >
          <div
            className="column is-half box"
          >
            <div
              className="section"
            >
              <h1
                className="title is-1"
              >
                Sign In
              </h1>
              <h3
                className="subtitle is-3"
              >
                Track your progress.
              </h3>
              <form
                onSubmit={[Function]}
              >
                <div
                  className="field"
                >
                  <div
                    className="control is-expanded"
                  >
                    <label
                      className="label"
                      htmlFor="email"
                    >
                      Email
                    </label>
                    <div
                      className="control"
                    >
                      <input
                        autoComplete="username"
                        className="input"
                        id="email"
                        name="email"
                        onBlur={[Function]}
                        onChange={[Function]}
                        type="email"
                        value=""
                      />
                    </div>
                  </div>
                </div>
                <div
                  className="field"
                >
                  <label
                    className="label"
                    htmlFor="password"
                  >
                    Password
                  </label>
                  <div
                    className="control"
                  >
                    <input
                      autoComplete="new-password"
                      className="input"
                      id="password"
                      name="password"
                      onBlur={[Function]}
                      onChange={[Function]}
                      type="password"
                      value=""
                    />
                  </div>
                </div>
                <div
                  className="field is-grouped"
                >
                  <div
                    className="control"
                  >
                    <button
                      className="button has-background-primary-dark "
                      disabled={false}
                      type="submit"
                    >
                      Sign In
                    </button>
                  </div>
                  <div
                    className="control"
                  >
                    <a
                      className="button is-link"
                      href="/signup"
                      onClick={[Function]}
                      onMouseEnter={[Function]}
                    >
                      Don't have an account? Sign Up.
                    </a>
                  </div>
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>
      <span
        className="tag credit"
      >
        Photo by  
        <a
          href="https://unsplash.com/@brett_jordan?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
        >
          Brett Jordan
        </a>
         on 
        <a
          href="https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
        >
          Unsplash
        </a>
      </span>
    </div>
  </section>,
]
`;
